xpanse/api/incident_management/v1/incidents.py

```diff
@@ -0,0 +1,25 @@
+from typing import Any, List
+
+from xpanse.endpoint import XpanseEndpoint
+from xpanse.iterator import XpanseResultIterator
+
+
+# TODO:// Implement Incidents https://jira-hq.paloaltonetworks.local/browse/EXPANDR-2596
+class IncidentsApi(XpanseEndpoint):
+    """
+    Part of the Alerts v2 Public API for handling Alerts Multi-Events v2.
+    See: https://docs-cortex.paloaltonetworks.com/r/Cortex-XPANSE/Cortex-Xpanse-API-Reference/Get-Incidents
+    See: https://docs-cortex.paloaltonetworks.com/r/Cortex-XPANSE/Cortex-Xpanse-API-Reference/Update-an-Incident
+    """
+
+    def list(self, **kwargs: Any) -> XpanseResultIterator:
+        raise NotImplementedError()
+
+    def get(self, incident_ids: List[str], **kwargs: Any) -> Any:
+        raise NotImplementedError()
+
+    def count(self, **kwargs: Any) -> int:
```

Comment:
The counts comes from the same endpoint as the "list all" endpoints - what are your thoughts on this count endpoint? We could make a request to the same endpoint, and return just the count with this?

Reply:
Yeah I think it already exists using `XpanseResultIterator.total`. My thought was to have a convenient method wrapping the `total_count` from the response so there's less of a chance of breaking existing usages of the sdk

Reply:
yep, sounds good 👍 we could note this, too - if this is used a lot, it may be worth making a separate endpoint for the counts so that we're not running the query 2 times when we just want the count (in the future)